repotracker/repotracker.go

```diff
@@ -814,30 +814,16 @@ func createVersionItems(ctx context.Context, v *model.Version, metadata VersionM
 			tasksToCreate = append(tasksToCreate, t)
 		}
 
-		var lastActivated *model.Version
 		activateAt := time.Now()
 		if metadata.TriggerID == "" && v.Requester != evergreen.AdHocRequester {
-			lastActivated, err = model.VersionFindOne(model.VersionByLastVariantActivation(projectInfo.Ref.Identifier, buildvariant.Name))
+			activateAt, err = projectInfo.Ref.GetActivationTime(&buildvariant)
 			if err != nil {
+				// should we be returning her?
```

Comment:
I don't think so, because that would prevent us from creating the version. IMO it's better to create it but not activate it in the case of an error.